index.js

```diff
@@ -23,6 +23,7 @@ function sniff(buffer, callback) {
 
         if (header.indexOf('\"tilejson\":') !== -1) return callback(null, 'tilejson');
         if ((header.indexOf('\"arcs\":') !== -1) || (header.indexOf('\"objects\":') !== -1)) return callback(null, 'topojson');
+        if ((header.indexOf('\"crs\":') !== -1) || (header.indexOf('\"features\":') !== -1)) return callback(null, 'geojson');
```

Comment:
Just a sanity check for myself, this says "if `crs` OR `features` exists in the string, pass it on as a geojson"? Just taking a look at the TopoJSON specification, which does not specify `crs` anywhere, but _it could_ be added by a user if they wanted: https://github.com/mbostock/topojson-specification/issues/5 - I'm not worried about this use case, though.